admin/scss/_preview.scss

```diff
@@ -277,7 +277,6 @@
 			@extend .icon-sprites-64x64;
 			@include sprite($sprites-64x64-preview);
 
-			display: block;
```

Comment:
The second param of the sprite mixin is the display value, if one is not given then it defaults to `display: block` making that line redundant (see https://github.com/silverstripe/silverstripe-framework/blob/master/admin/scss/_spritey.scss#L184).